files/zh-cn/web/api/navigator/wakelock/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Navigator: wakeLock property"
+slug: Web/API/Navigator/wakeLock
+l10n:
+  sourceCommit: 0d9c7bb3574c48373ad96e2efc6701f306a9a3af
+---
+
+{{ApiRef("Screen Wake Lock API")}}{{securecontext_header}}
+
+**`wakeLock`** 属性是 {{domxref("Navigator")}} 接口的只读属性，它返回一个 {{DOMxRef("WakeLock")}} 接口，允许文档获取屏幕唤醒锁定。当屏幕唤醒锁定处于活动状态时，用户代理将尝试阻止设备调暗屏幕、完全关闭屏幕或显示屏幕保护程序。
```

Comment:
- Let's highlight what the property does rather than which type of property it is:
- Screen Wake Lock => 屏幕唤醒锁, see also: https://www.w3.org/zh-hans/news/2015/first-public-working-draft-wake-lock-api-draft-published/

```suggestion
{{domxref("Navigator")}} 接口的 **`wakeLock`** 只读属性返回一个允许文档获取屏幕唤醒锁的 {{DOMxRef("WakeLock")}} 接口。当屏幕唤醒锁处于激活状态时，用户代理将尝试阻止设备调暗屏幕、完全关闭屏幕或显示屏幕保护程序。
```